week5/5.3/whisper.py

```diff
@@ -0,0 +1,35 @@
+"""
+whisper.
+By: Eliad Karni.
+The feature decode messages which contained in the file "logo.jpg".
+Used the help of the websites:
+Binary read -> https://pythonguides.com/python-read-a-binary-file/
+Ignore decode errors -> https://docs.python.org/3/library/codecs.html
+"""
+
+
+def get_a_whisper() -> str:
+    """
+    The function is a iterate generator which decode a whisper from the file "logo.jpg".
+    1 message each time.
+    :return: a decoded word from the file "logo.jpg".
+    """
+    with open('logo.jpg', 'br') as file_reader:
+        decoded_word = ''
+        char = file_reader.read(1)
+        while char:
```

Comment:
If im not mistakes, you can replace this while loop with a for character in file_reader.read()  - it will return the whole file so youll be able to iterate over the characters - its less memory efficient but will produce a shorted and more readable code